docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,221 @@
+# Create Your First Tizen IoT .NET Application
+
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Tizen .NET application development process. With the instructions, you can create and run a basic .NET application, which displays some texts on the screen without any user interaction.
+
+1.  Before you get started with developing Tizen applications, set up the [development environment](../../../vstools/install.md).
+
+2. [Create a Project](#create-a-project) using Visual Studio.
+
+    This step shows how you can use a predesigned project template that creates all the basic files and folders required for your project.
+
+3. [Build Your Application](#build-your-application).
+
+    After you have implemented code for the features you want, this step shows how you can build the application to validate and compile
+    the code.
```

Comment:
@xbldx 

Line 13, although the line renders properly on GitHub, but can you just check once why "the code" is in the next line?
Just verify, if it is fine, kindly ignore.

Reply:
Fixed it.